content/de/docs/contribute/participate/pr-wranglers.md

```diff
@@ -0,0 +1,81 @@
+
+### Hilfreiche GitHub-Anfragen f&uuml;r Wranglers
+
+Die folgenden Anfragen sind beim Wrangling hilfreich.
+Wenn du diese Anfragen abgearbeitet hast, ist die verbleibende Liste der zu pr&uuml;fenden PRs meist klein.
+Diese Anfragen schlie&szlig;en Lokalisierungs-PRs aus. Alle Anfragen beziehen sich auf den Hauptast, au&szlig;er der letzten.
+
+- [Kein CLA, nicht zusammenf&uuml;rbar](https://github.com/kubernetes/website/pulls?q=is%3Aopen+ist%3Apr+label%3A%22cncf-cla%3A+no%22+-label%3A%22do-not-merge%2Fwork-in-progress%22+-label%3A%22do-not-merge%2Fhold%22+label%3Alanguage%2Fen):
+  Erinnere den Beitragenden daran, den CLA zu unterschreiben. Wenn sowohl der Bot als auch ein Mensch sie daran erinnert haben, schlie&szlig;e
+  den PR und erinnere die Autoren daran, dass sie ihn erneut &ouml;ffnen k&ouml;nnen, nachdem sie den CLA unterschrieben haben.
+  **&Uuml;berpr&uuml;fe keine PRs, deren Autoren den CLA nicht unterschrieben haben!**
+- [Ben&ouml;tigt LGTM](https://github.com/kubernetes/website/pulls?q=is%3Aopen+ist%3Apr+-label%3A%22cncf-cla%3A+kein%22+-label%3Ado-not-merge%2Fwork-in-progress+-label%3Ado-not-merge%2Fhold+label%3Alanguage%2Fen+-label%3Algtm):
+  Listet PRs auf, die ein LGTM von einem Mitglied ben&ouml;tigen. Wenn der PR eine technische &Uuml;berpr&uuml;fung ben&ouml;tigt, schalte einen der vom Bot vorgeschlagenen Reviewer ein. Wenn der Inhalt &uuml;berarbeitet werden muss, f&uuml;ge Vorschl&auml;ge und Feedback in-line hinzu.
+- [Hat LGTM, braucht die Zustimmung von Docs](https://github.com/kubernetes/website/pulls?q=is%3Aopen+is%3Apr+-label%3Ado-not-merge%2Fwork-in-progress+-label%3Ado-not-merge%2Fhold+label%3Alanguage%2Fen+label%3Algtm+):
+  Listet PRs auf, die einen `/approve`-Kommentar ben&ouml;tigen, um zusammengef&uuml;hrt zu werden.
+- [Quick Wins](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+base%3Amain+-label%3A%22do-not-merge%2Fwork-in-progress%22+-label%3A%22do-not-merge%2Fhold%22+label%3A%22cncf-cla%3A+yes%22+label%3A%22size%2FXS%22+label%3A%22language%2Fen%22): Listet PRs gegen den Hauptzweig auf, die nicht eindeutig blockiert sind. (&auml;ndere "XS" in der Gr&ouml;&szlig;enbezeichnung, wenn du dich durch die PRs arbeitest [XS, S, M, L, XL, XXL]).
+- [Nicht gegen den Hauptast](https://github.com/kubernetes/website/pulls?q=is%3Aopen+ist%3Apr+label%3Alanguage%2Fen+-base%3Amain): Wenn der PR gegen einen `dev-`Ast gerichtet ist, ist er f&uuml;r eine kommende Ver&ouml;ffentlichung. Weise diesen dem [Docs Release Manager](https://github.com/kubernetes/sig-release/tree/master/release-team#kubernetes-release-team-roles) zu: `/assign @<manager's_github-username>`. Wenn der PR gegen einen alten Ast gerichtet ist, hilf dem Autor herauszufinden, ob er auf den richtigen Ast gerichtet ist.
+
+### Hilfreiche Prow-Befehle f&uuml;r Wranglers
```

Comment:
I would avoid HTML entities in headings. That's a recipe for trouble if you ask me, because headings become fragment identifiers.